src/main/java/com/jlr/ttl/poc/loadmanager/models/Vehicle_Load.java

```diff
@@ -0,0 +1,61 @@
+package com.jlr.ttl.poc.loadmanager.models;
+
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "vehicle_loads")
+public class Vehicle_Load {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column
+    private Integer id;
+
+    @Column(name = "veh")
+    private Integer vehicle_id;
+
+    @Column
+    private String load_ref;
+
+    @Column(name = "end_date")
+    private Date date;
+
+    public Vehicle_Load() {
```

Comment:
Remove constructors and getters and setters. Use the following annotations instead:

AllArgsConstructor
NoArgsConstrutctor
Data
Build